src/Tizen.NUI/src/public/WebView/WebBackForwardList.cs

```diff
@@ -36,9 +36,9 @@ protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef
         }
 
         /// <summary>
-        /// Get uri.
+        /// Get url.
```

Comment:
when used in text: url -> URL

Reply:
Done.
I updated it in another patch https://github.com/Samsung/TizenFX/pull/3520